src/css/controls/imports/slider.less

```diff
@@ -88,49 +90,20 @@
         width: @def-rail-width;
     }
 
-    .jw-slider-container {
-        height: @volume-rail-length;
-    }
-
     .jw-knob {
-        right: 0;
-        left: (@def-rail-width - @thumb-size) / 2;
-        margin-bottom: @thumb-size/-2;
+        left: -50%;
+        margin-bottom: (-@thumb-size / 2);
     }
 }
 
 .jw-slider-time {
     width: 100%;
```

Comment:
`jw-slider-time` needs left+right padding so that the slider knob doesn't go off the player and to make it usable on touch devices.

Before:
<img width="439" alt="screen shot 2017-07-31 at 2 54 46 pm" src="https://user-images.githubusercontent.com/4603952/28793131-0809f170-7600-11e7-8a8e-eb3539052a49.png">

After adding 14px:
<img width="443" alt="screen shot 2017-07-31 at 2 56 49 pm" src="https://user-images.githubusercontent.com/4603952/28793218-533b17a0-7600-11e7-8425-92d858ce85fd.png">